app/views/posts/show.html.erb

```diff
@@ -1,12 +1,23 @@
-<div class="detail full-width">
+<div class="detail full-width d-margin">
     <div class="post-head">
       <h3>Post  #<%= @post.id %> by <%= @user.name %></h3>
       <span>Comments: <%= @post.comments_counter || 0 %> Likes:<%= @post.likes_counter || 0 %></span>
     </div>
     <p class="comments"><%= @post.text%></p>
 </div>
-<div class="detail full-width d-margin">
+
+<div class="detail full-width like-comment">
+  <%= link_to new_comment_path(@post.id) do %>
+    <button>Add New Comment</button>
+  <% end %>
+
+  <%= link_to new_like_path() do %>
+    <button>Like</button>
+  <% end %>
```

Comment:
- **[OPTIONAL]** It would be nice to add the `create` like feature directly on this button without having to move to a new like page to create a like. We can click this button and call the `create` method in the likes controller without having to navigate to a new page. This way we don't need to the `new` method in the likes controller.